notebooks/example_container_fmriprep_4review.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example config yaml file for fMRIPrep version 20.2.3.
+
+# Warning!!!
+#   This is only an example, which may not necessarily fit your purpose,
+#   or be an optimized solution for your case,
+#   or be compatible to the fMRIPrep version you're using.
+#   Therefore, please change and tailor it for your case before use it!!!
+
+# Arguments when executing the BIDS App using `singularity run`:
+# You should not have flags of : `--participant-label`, or `--bids-filter-file`!
+babs_singularity_run:
+    -w: "$BABS_TMPDIR"   # this is a placeholder. To be changed to `${PWD}/.git/tmp/wkdir`
+    --n_cpus: '1'
+    --stop-on-first-crash: ""
+    --fs-license-file: "$FREESURFER_LICENSE" # this is a placeholder.
+    --skip-bids-validation: Null  # Null or NULL is also a placeholder
+    --output-spaces: MNI152NLin6Asym:res-2
```

Comment:
Do you want to allow a list here, or should this be a space-separated string? Maybe having multiple output spaces in the example YAML would make things clearer.

Reply:
Oh, you mean for the case like `--output-spaces MNI152NLin6Asym:res-2 MNI152NLin2009cAsym`? Thanks for pointing this possibility out! I did not know this.

I can double check if `--output-spaces: 'MNI152NLin6Asym:res-2 MNI152NLin2009cAsym'` works, and add it to either example YAML file or in the docs.